templates/frontend/components/header.tpl

```diff
@@ -78,6 +68,9 @@
 					{load_menu name="primary" id="navigationPrimary" ulClass="pkp_navigation_primary"}
 				{/capture}
 
+				<nav class="cmp_skip_to_content">
+					{include file="frontend/components/skipLinks.tpl"}
+				</nav>
```

Comment:
Move this `<nav>` block to before the `{capture}` block a few lines above. This will keep all of the primary nav code together.

Reply:
Done!

Reply:
Ah, I've just noticed that the wrapper `<nav>` has been moved out of `skipLinks.tpl`. I think we need to put it back there.

So here in this file (`header.tpl`) we should just have:

```
{include file="frontend/components/skipLinks.tpl"}
```

Then, in `templates/frontend/components/skipLinks.tpl` the `<nav>` wrapper should be put back:

```
<nav class="cmp_skip_to_content">
  ...
</nav>
```

Reply:
Check now?